ow_gazebo_plugins/src/BalovnevModelPlugin/BalovnevModelPlugin.cpp

```diff
@@ -205,23 +211,20 @@ void BalovnevModelPlugin::resetDepth() {
   m_pub_depth.publish<std_msgs::Float64>(depth);
 }
 
-bool BalovnevModelPlugin::isScoopDigging()
+bool BalovnevModelPlugin::isScoopDigging() const
 {
   static const Vector3 SCOOP_DOWNWARD(0.0, 0.0, 1.0);
-  static const Vector3 WORLD_DOWNWARD(0.0, 0.0, -1.0);
-  Vector3 scoop_bottom(
-    m_link->WorldPose().Rot().RotateVector(SCOOP_DOWNWARD)
-  );
+  Vector3 scoop_bottom(m_link->WorldPose().Rot().RotateVector(SCOOP_DOWNWARD));
   return WORLD_DOWNWARD.Dot(scoop_bottom) > 0.0;
 }
 
 void BalovnevModelPlugin::onModDiffVisualMsg(
   const modified_terrain_diff::ConstPtr &msg)
 {
   // DEBUG CODE
-  static int count_called = 0;
-  count_called++;
-  static int count_finished = 0;
+  // static int count_called = 0;
+  // count_called++;
+  // static int count_finished = 0;
```

Comment:
Do we need to keep this commented out debug code? Would an ifdef serve better?

Reply:
Removed. Other debug code has been left in because I intend to use it for the next PR on this epic branch.